compute/Dockerfile.compute-node

```diff
@@ -366,7 +366,7 @@ FROM build-deps AS rum-pg-build
 ARG PG_VERSION
 COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
 
-COPY patches/rum.patch /rum.patch
+COPY compute/patches/rum.patch /rum.patch
```

Comment:
It's a bit strange to see that paths are not relative to Dockerfile location. Not sure, but maybe it could be solved by specifying the right path / workdir in the image building jobs?

Reply:
I agree it's a bit strange. I don't see a better way though. The Dockerfile needs to have the git root as the build context, as it builds things from `compute_tools` and `libs` too.

It might be a good idea to rearrange things more, and move `compute_tools` as `compute/compute_ctl`. `compute_ctl` is clearly part of what goes into a compute image, so it'd make sense to have it in the `compute/` subdir. However, it also relies on stuff in `libs/`, and the `Cargo.*` files in the root, and probably other things I've missed.